### PR TITLE
Make `compact` the default Input.ChoiceSet style

### LINKED ISSUE #2446: WPF Input.ChoiceSet incorrectly defaults to "expanded"
# Platform

* .NET WPF

# Version of SDK

Latest master as of 2/21/2019 (so 1.2 feature set).

Also 1.1 behaved this way too (just cloned 1.1 branch and tried WPF Visualizer), so this is NOT a new regression

Also 1.0 behaved this way too... so since the very beginning it's been like this.

# Issue

`Input.ChoiceSet` is supposed to default to `"style": "compact"`. However, in the WPF visualizer, when style isn't specified, it's displayed as expanded...

### Expected

image

### Actual

image

```json
{
    "type": "AdaptiveCard",
    "body": [
        {
            "type": "TextBlock",
            "size": "Medium",
            "weight": "Bolder",
            "text": "\"style\": undefined"
        },
        {
            "type": "Input.ChoiceSet",
            "id": "myChoiceSet",
            "value": "Choice 1",
            "choices": [
                {
                    "title": "Choice 1",
                    "value": "Choice 1"
                },
                {
                    "title": "Choice 2",
                    "value": "Choice 2"
                }
            ]
        }
    ],
    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
    "version": "1.0"
}
```

### PR DESCRIPTION
**Pull request by paulcam206**
(to align with other platforms)

Fixes #2446

## How verified
* existing unit tests
* sample card from linked issue in visualizers

### CODE REVIEW

**Comment by matthidinger** [comment]
@paulcam206 can we make sure to doc this breaking change in the .NET SDK? We can include a workaround to maintain compat between versions by explicitly including the `style` property in the payload

**Comment by shalinijoshi19** [comment]
Let's try to include "How verified" in the Description

File: source/dotnet/Library/AdaptiveCards/AdaptiveChoiceSetInput.cs [hunk 1]
```diff
@@ -40,6 +40,7 @@ public class AdaptiveChoiceSetInput : AdaptiveInput
 #if !NETSTANDARD1_3
         [XmlAttribute]
 #endif
+        [DefaultValue(typeof(AdaptiveChoiceInputStyle), "compact")]
```
Comment (shalinijoshi19) [inline source/dotnet/Library/AdaptiveCards/AdaptiveChoiceSetInput.cs]: Do we have no unit tests covering our default options?

**Comment by shalinijoshi19** [comment]
Who uses this btw outside of Bot Framework? 

---
In reply to: [486864058](https://github.com/Microsoft/AdaptiveCards/pull/2765#issuecomment-486864058) [](ancestors = 486864058)

**Comment by paulcam206** [comment]
This is now noted in our OneNote breaking changes page

File: source/dotnet/Library/AdaptiveCards/AdaptiveChoiceSetInput.cs (hunk 1 above)
Comment (paulcam206) [reply source/dotnet/Library/AdaptiveCards/AdaptiveChoiceSetInput.cs]: They currently cover roundtripping, which means that this type of failure isn't caught. I've created a `Known Gaps` section in the OneNote to document this.

**Pull request merged**